packages/cli/src/cmds/agentInstaller/gradleInstaller.ts

```diff
@@ -111,7 +111,7 @@ export default class GradleInstaller
     const lines = parseResult.plugins
       ? buildFileSource
           .substring(parseResult.plugins.lbrace + 1, parseResult.plugins.rbrace)
-          .split('\n')
+          .split(os.EOL)
```

Comment:
I don't think that's right. Config files can have Unix line breaks even on Windows. We should handle this case and make sure we leave the existing line breaks as we found them, if only to limit diff noise. (I think it's acceptable, if a bit fugly, to insert any new ones as `os.EOL`.)

Reply:
Yeah, this is wrong. Thanks.